### ISSUE TITLE
Cancel does not stop the uploading of DTSAPI texts

### ISSUE THREAD

**Issue by monzug** [alpheios-project/alignment-editor-new#517]
related to #473 
when uploading a DTSAPI text and click on Cancel, the cancel action does not stop the uploading but it should.

**Comment by irina060981** [comment]
It is not so easy to implement such behaviour as it seems to be - need some time to think about it.

**Comment by monzug** [comment]
we can wait after Pietro's release

**Comment by irina060981** [comment]
@monzug , I suggest to simplify this issue - 
let's disable buttons until uploading process is doing and then we prevent a user from this problem

Any other solution will need a serious refactoring:
- uploading could not be cancelled
- that's why on cancel and open again I would need to track two requests (or more) and examine what results should be shown and what should be forgotten

I believe the time that would be spent for this complexity doesn't worth this rare case. And it is not really influence on other application aspect.

What do you think?

**Comment by irina060981** [comment]
>solution has been implemented. How about closing the pop-up while the file is still uploading? right now, it does still upload in the background. I can resume a json file then go back to DTSAPI and the pop-up is still uploading.

As I described before - there is no way to stop file uploading.
So there is no easy solution for the described case.

I don't think that this bug is worth creating a complex workflow for tracking parallel background process to hide it.
I suggest to leave it as is.
Do you agree, Monica?

**Comment by monzug** [comment]
Absolutely. As you noticed as been verified. How can we keep track of this issue for future reference?
